### DISCUSSION TITLE
How to change default container log file size

### DISCUSSION THREAD

**Discussion by AvniWala-AWN** [k3s-io/k3s#3401] [Q&A] [answered]
Dear All,

Would appreciate inputs on how to modify the default container log file size in k3s?

I tried to use --kubelet-arg and supply container-log-max-size  (https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/). But it looks like that option is deprecated. 

I do _not_ want to specify a full config file using  "--config string" option since that would require me to override all the default values.

Any other suggestions please?

Best Regards,
--Avni

**Comment by brandond** [comment, marked as answer]
All kubelet CLI flags are technically deprecated, but most folks still use them. You can pass flags to the kubelet via the k3s `--kubelet-arg=flag=value`.
